spec/requests/api/cycles_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'swagger_helper'
+require 'rails_helper'
+
+describe 'Cycles API' do
+  let(:user) { create(:user) }
+
+  before do |response|
+    sign_in user unless response.metadata[:skip_before]
+  end
+
+  path '/api/curriculums/{curriculum_id}/cycles' do
+    get 'Returns all Cycles' do
+      tags 'Cycles'
+      description "Retrieves a list of all curriculum's cycles"
+      parameter name: :curriculum_id, in: :path, type: :string
+      produces 'application/json'
+      operationId 'getCycles'
+
+      let(:curriculum_id) { create(:curriculum).id }
+      let(:cycle) { create(:cycle, curriculum_id:) }
+
+      response '200', 'Success' do
+        schema type: :array,
+               items: {
+                 type: :object,
+                 properties: {
+                   id: { type: :integer },
+                   name: { type: :string },
+                   learning_goals: { type: :text },
+                   challenge_description: { type: :text },
```

Comment:
`:text` is not a valid type recognized by rswag. Why don't you use :string?
<img width="490" alt="image" src="https://user-images.githubusercontent.com/17869861/197200872-78ea0095-8a05-48ce-9bd3-56e2c26122fd.png">

Reply:
You're right, I used the Rails type. I'm changing it.